### ISSUE TITLE
Apollo cannot autodrive the car in simulator

### ISSUE THREAD

**Issue by umdreamer** [lgsvl/simulator#1712]
I have installed the svlsimualtor 2021.2, and Apollo 6.0 in Ubuntu 16.04 with Nvidia driver 450 with CUDA 11. Everything seems OK according to the documentation: https://www.svlsimulator.com/docs/system-under-test/apollo-master-instructions/.

But in the svl and apollo simulation, the car stopped, and could not go automatically. It seems that I cannot designate the starting point and ending point of the car. But how to do it in Apollo Dreamview pannel? It didn't work after I clicked the mouse.
![image](https://user-images.githubusercontent.com/15782682/130613202-03c618a6-f665-4960-9d6d-6a2d8645b90a.png)

**Comment by EricBoiseLGSVL** [comment]
@revati-naik @lemketron ?

**Comment by revati-naik** [comment]
@umdreamer Please make sure you turn on the modules in Dremaview under the tab "Module Controller". Follow the instructions given under [Launching Apollo alongside the Simulator](https://www.svlsimulator.com/docs/system-under-test/apollo-master-instructions/). 

Also, please make sure you give a valid destination point. For example, start with placing a destination point along the same lane-line (green line) as the ego vehicle.

**Comment by umdreamer** [comment]
@revati-naik Yes, I have done according to the documentation. Here are some snapshots of my program.
![image](https://user-images.githubusercontent.com/15782682/130708096-b0049c38-c84a-446b-9364-520867582601.png)
![image](https://user-images.githubusercontent.com/15782682/130708170-1f78d293-6603-492e-9fd4-cddd397d2b4f.png)
![image](https://user-images.githubusercontent.com/15782682/130708202-2fc7e8d5-e174-4038-9193-da690506f3c9.png)
There are two points I placed. Are they correct?
![image](https://user-images.githubusercontent.com/15782682/130709673-9155c21c-d079-4a72-99b9-857241be8073.png)

**Comment by lemketron** [comment]
> There are two points I placed. Are they correct?

Please click `Remove All Points` and then click only once to set a destination directly in front of the ego vehicle.

As noted in the Apollo Dreamview documentation, you should see a red line indicating the route (after you click to set a valid destination and then click "Send Routing Request" if the `Routing` module is enabled) and then (if the `Planning` module is enabled) you should also see a "green thick curvy band [which] indicates the planned trajectory". At that point, if there is no stop fence in front of the ego vehicle, then if the `Control` module is enabled, the ego vehicle should start driving. Sometimes hitting the up arrow on your keyboard to nudge the car a bit can also help (and will ensure that the vehicle IS able to move).

**Comment by umdreamer** [comment]
@lemketron I made it. It works now. Thank you very much.
